external/fv3fit/tests/test_combined_output_model.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+import os
+import pytest
+import xarray as xr
+import yaml
+
+
+from fv3fit._shared.models import CombinedOutputModel
+from fv3fit.testing import ConstantOutputPredictor
+
+
+def test_CombinedOutputModel():
+    model0 = ConstantOutputPredictor(
+        input_variables=["in0", "in1"], output_variables=["out0a", "out0b"]
+    )
+    model1 = ConstantOutputPredictor(
+        input_variables=["in1", "in2"], output_variables=["out1a", "out1b"]
+    )
+
+    model0.set_outputs(out0a=np.ones(10), out0b=np.ones(10))
+    model1.set_outputs(out1a=np.ones(10) * 2, out1b=np.ones(10) * 2)
+
+    da = xr.DataArray(data=np.ones((5, 10)), dims=["x", "z"])
+    X = xr.Dataset({"in0": da, "in1": da, "in2": da})
+
+    combined_model = CombinedOutputModel([model0, model1])
+    combined_prediction = combined_model.predict(X)
+    np.testing.assert_array_equal(
+        combined_prediction["out0a"].values, model0.predict(X)["out0a"]
+    )
+    np.testing.assert_array_equal(
+        combined_prediction["out1a"].values, model1.predict(X)["out1a"]
+    )
```

Comment:
mostly out of curiosity, why do you need `.values` for the combined predictions but not for the individual model predictions?

Reply:
No reason, cleaned it up.